README.md

```diff
@@ -696,7 +696,7 @@ An open-source autonomous AI framework to enable development and deployment auto
 
 ### Description
 - An AI agent framework
-- Open source, but infrastructure is closed-source
+- Open source, but infrastructure is -source
```

Comment:
Could you please provide more explanation on the changes in this line?

Reply:
Should be reverted. Sorry, an issue on my side